lib/cli.rb

```diff
@@ -10,16 +18,22 @@
     puts opts
     exit
   end
+  opts.on("-a name", "--adapter=name", "Run with datadase or CSV") do |v|
+    if v == "DB"
+      conn = PG::Connection.open( dbname: 'monmon', user: 'postgres')
+      table = convert_keys(conn.exec("SELECT * from accounts"))
+    elsif v == "CSV"
+      i = 0
+      CSV.foreach(options[:file], headers: true, header_converters: :symbol) do |row|
+        puts("#{row[:type]}: #{row[:name]}:  #{row[:currency]}\t - #{row[:amount]}  #{row[:currency]}")
+        table[i] = {type: row[:type], name: row[:name], currency: row[:currency], amount: row[:amount]}
+        i = i+1
+      end
+    else
+      abort "Error!"
```

Comment:
The program should explain what kind of error it is. Otherwise, it's not clear what's wrong and it's not clear how to fix this problem without looking into code.